src/widgets/histogram/content-view.js

```diff
@@ -332,7 +333,7 @@ module.exports = cdb.core.View.extend({
     var $tooltip = this.$('.js-tooltip');
 
     if (info && info.data) {
-      var bottom = this.defaults.chartHeight + 3 - info.top;
+      var bottom = this.defaults.chartHeight + 23 - info.top;
```

Comment:
Why this magic number? I'd change it by a constant or something to clearly represent the intention of the code.